components/arm/arm.go

```diff
@@ -66,8 +67,8 @@ const (
 // and there are joints which are out of bounds.
 const MTPoob = "cartesian movements are not allowed when arm joints are out of bounds"
 
-var defaultArmPlannerOptions = map[string]interface{}{
-	"motion_profile": motionplan.LinearMotionProfile,
+var defaultArmPlannerOptions = &motionpb.Constraints{
+	LinearConstraint: []*motionpb.LinearConstraint{{}},
```

Comment:
extranneous braces

Reply:
These are not extraneous. That's the difference between "list of length zero", and "list of length one, whose only member is a bare-instantiated struct with no values filled in". If you remove them you will get radically different behavior.

I'll actually type that though to make it more readable.